src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/message/processor/CourtCaseEventsProcessor.kt

```diff
@@ -48,18 +46,13 @@ class CourtCaseEventsProcessor(
 
       else -> {
         log.debug("Received case type ${UNKNOWN.name}")
-        telemetryService.trackEvent(UNKNOWN_CASE_RECEIVED, emptyMap())
```

Comment:
As above, no need for an AppInsights event and I don't think this actually happens in practice in any case.